hera_qm/xrfi.py

```diff
@@ -2008,7 +2014,7 @@ def xrfi_run(ocalfits_files=None, acalfits_files=None, model_files=None, data_fi
     flags = []
     # medfilt -> meanfilt.
     cal_algs = ['detrend_meanfilt', 'detrend_meanfilt', 'zscore_full_array']
-    labels = [' gains, mean filter.', ' chisq, mean filter.', ' Renormalized chisq, median filter, round 2.']
+    labels = [' gains, mean filter.', ' chisq, mean filter.', ' overall z-score, median filter, round 2.']
```

Comment:
One minor comment: I think it's much clearer if you say that it's the overall z-score of abscal/omnical chi^2. Overall z-score is vague.

Reply:
I've gone and fixed this myself. If you're OK with my change, feel free to merge.

Reply:
I'm good with this change.